diff --git a/node_modules/@lerna/command/index.js b/node_modules/@lerna/command/index.js
index 2a29f76..64b0263 100644
--- a/node_modules/@lerna/command/index.js
+++ b/node_modules/@lerna/command/index.js
@@ -34,7 +34,8 @@ class Command {
 
     if (!this.composed) {
       // composed commands have already logged the lerna version
-      log.notice("cli", `v${argv.lernaVersion}`);
+      // Comment this out, because it is annoying when running multiple lerna commands in parallel
+      // log.notice("cli", `v${argv.lernaVersion}`);
     }
 
     // launch the command
@@ -250,7 +251,8 @@ class Command {
   runPreparations() {
     if (!this.composed && this.project.isIndependent()) {
       // composed commands have already logged the independent status
-      log.info("versioning", "independent");
+      // Comment this out, because it is annoying when running multiple lerna commands in parallel
+      // log.info("versioning", "independent");
     }
 
     if (!this.composed && this.options.ci) {
