diff --git a/node_modules/haul/src/utils/makeReactNativeConfig.js b/node_modules/haul/src/utils/makeReactNativeConfig.js
index 0cf0289..1a6c33b 100644
--- a/node_modules/haul/src/utils/makeReactNativeConfig.js
+++ b/node_modules/haul/src/utils/makeReactNativeConfig.js
@@ -374,7 +374,7 @@ function makeReactNativeConfig(
 function injectPolyfillIntoEntry({
   entry: userEntry,
   root,
-  initializeCoreLocation = 'node_modules/react-native/Libraries/Core/InitializeCore.js',
+  initializeCoreLocation = 'react-native/Libraries/Core/InitializeCore.js',
   dev = true,
   disableHotReloading = false,
 }: {
@@ -386,7 +386,7 @@ function injectPolyfillIntoEntry({
 }) {
   const reactNativeHaulEntries = [
     ...getPolyfills(),
-    require.resolve(path.join(root, initializeCoreLocation)),
+    require.resolve(initializeCoreLocation),
   ];
 
   if (dev && !disableHotReloading) {
